### PR TITLE
SEOImages: User Story 1463880, Part 27

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880)

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032000018/Report/

File: docs/framework/winforms/controls/bindingnavigator-control-overview-windows-forms.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can use the <xref:System.Windows.Forms.BindingNavigator> control to create a
 ## How the BindingNavigator Works  
  The <xref:System.Windows.Forms.BindingNavigator> control is composed of a <xref:System.Windows.Forms.ToolStrip> with a series of <xref:System.Windows.Forms.ToolStripItem> objects for most of the common data-related actions: adding data, deleting data, and navigating through data. By default, the <xref:System.Windows.Forms.BindingNavigator> control contains these standard buttons. The following screen shot shows the <xref:System.Windows.Forms.BindingNavigator> control on a form.
```
Comment (mairaw) [inline docs/framework/winforms/controls/bindingnavigator-control-overview-windows-forms.md]:
```suggestion

 The <xref:System.Windows.Forms.BindingNavigator> control is composed of a <xref:System.Windows.Forms.ToolStrip> with a series of <xref:System.Windows.Forms.ToolStripItem> objects for most of the common data-related actions: adding data, deleting data, and navigating through data. By default, the <xref:System.Windows.Forms.BindingNavigator> control contains these standard buttons. The following screenshot shows the <xref:System.Windows.Forms.BindingNavigator> control on a form:
```

File: docs/framework/winforms/controls/bindingnavigator-control-overview-windows-forms.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ You can use the <xref:System.Windows.Forms.BindingNavigator> control to create a
 ## How the BindingNavigator Works  
  The <xref:System.Windows.Forms.BindingNavigator> control is composed of a <xref:System.Windows.Forms.ToolStrip> with a series of <xref:System.Windows.Forms.ToolStripItem> objects for most of the common data-related actions: adding data, deleting data, and navigating through data. By default, the <xref:System.Windows.Forms.BindingNavigator> control contains these standard buttons. The following screen shot shows the <xref:System.Windows.Forms.BindingNavigator> control on a form.  
   
- ![BindingNavigator Control](./media/cpdatacontainerctrl.gif "cpDataContainerCtrl")  
+ ![Screenshot showing the BindingNavigator Control.](./media/bindingnavigator-control-overview-windows-forms/bindingnavigator-control-form.gif)
```
Comment (mairaw) [inline docs/framework/winforms/controls/bindingnavigator-control-overview-windows-forms.md]:
```suggestion
 ![Screenshot showing the BindingNavigator control.](./media/bindingnavigator-control-overview-windows-forms/bindingnavigator-control-form.gif)  
```

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 3]
```diff
@@ -220,11 +220,11 @@ The localization process begins after the unlocalized `MyDialog.resources.dll` f
   
  **English:**  
   
- ![English page](./media/englishhomepage.jpg "EnglishHomepage")  
+ ![Screenshot showing an English Homepage.](./media/wpf-globalization-and-localization-overview/english-homepage-sample.jpg)  
   
  **Arabic:**  
   
- ![Arabic page](./media/arabichomepage.jpg "ArabicHomepage")  
+ ![Screenshot showing an Arabic Homepage.](./media/wpf-globalization-and-localization-overview/arabic-homepage-sample.jpg)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]: home page - two words (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/h/home-home-page)

```suggestion
 ![Screenshot showing an Arabic home page.](./media/wpf-globalization-and-localization-overview/arabic-home-page-sample.jpg)  
```

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 4]
```diff
@@ -220,11 +220,11 @@ The localization process begins after the unlocalized `MyDialog.resources.dll` f
   
  **English:**  
   
- ![English page](./media/englishhomepage.jpg "EnglishHomepage")  
+ ![Screenshot showing an English Homepage.](./media/wpf-globalization-and-localization-overview/english-homepage-sample.jpg)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]:
```suggestion
 ![Screenshot showing an English home page.](./media/wpf-globalization-and-localization-overview/english-home-page-sample.jpg)  
```

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 5]
```diff
@@ -220,11 +220,11 @@ The localization process begins after the unlocalized `MyDialog.resources.dll` f
...
   
- ![English page](./media/englishhomepage.jpg "EnglishHomepage")  
+ ![Screenshot showing an English Homepage.](./media/wpf-globalization-and-localization-overview/english-homepage-sample.jpg)  
   
  **Arabic:**  
   
- ![Arabic page](./media/arabichomepage.jpg "ArabicHomepage")  
+ ![Screenshot showing an Arabic Homepage.](./media/wpf-globalization-and-localization-overview/arabic-homepage-sample.jpg)  
   
 ### Designing a Global Microsoft Homepage
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]:
```suggestion
### Designing a global Microsoft home page  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few comments.

**Comment by damabe** [comment]
@mairaw Changes are incorporated. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019040200056/Report/

**Review by mairaw — APPROVED** [review]
Thanks! :shipit:

**Pull request merged**